### DISCUSSION TITLE
TextArea component is missing from react-aria-components

### DISCUSSION THREAD

**Discussion by eashish93** [adobe/react-spectrum#4594] [General]
Playing with react-aria-components for a mvp project and found that `TextArea` component is missing from the doc. In the meantime I'm using my own created with hooks version. It's strange it's not in alpha because it's literally a copy of `Input` element.

**Comment by snowystinger** [comment]
Looks like it got missed, it's still early days of React Aria Components. Thanks for alerting us. I've opened an issue for it https://github.com/adobe/react-spectrum/issues/4595
